Gemfile.lock

```diff
@@ -288,10 +288,11 @@ GEM
 
 PLATFORMS
   x86_64-darwin-19
+  x86_64-darwin-20
 
 DEPENDENCIES
   cocoapods
   fastlane
 
 BUNDLED WITH
-   2.2.3
+   2.2.6
```

Comment:
What's the purpose of updating the bundler?

Reply:
Wrong account lmao